README.md

```diff
@@ -8,6 +8,11 @@ easy-to-use client library for the [Google Ads API][].
 
 # Documentation for Users
 
+**For thorough documentation of all features and utilities, visit our
+[Developers Site ruby guide][].**
+
+[Developers Site ruby guide]: https://developers.google.com/google-ads/api/docs/client-libs/ruby/
+
 ## Getting started
 
 `google-ads-googleads` will allow you to connect to the
```

Comment:
GitHub won't let me comment on unchanged lines, but I noticed the wording of "In order to achieve so" on line 16 is a bit awkward. Could you rephrase that?

Reply:
Removed that whole sentence. It didn't seem to add much.

Reply:
Sorry, I wrote that comment before I realized the link style you're using.